Sources/TensorFlow/Optimizer.swift

```diff
@@ -202,14 +202,15 @@ public class AdaMax<Model: Layer>: Optimizer
 
 /// RMSProp optimizer.
 ///
-/// It is recommended to leave the parameters of this optimizer at their default values (except the
-/// learning rate, which can be freely tuned). This optimizer is usually a good choice for recurrent
-/// neural networks.
+/// It is recommended to leave the parameters of this optimizer at their default values (except for 
+/// the learning rate, which can be freely tuned). This optimizer is usually a good choice for 
+/// recurrent neural networks.
 ///
-/// Reference: ["rmsprop: Divide the gradient by a running average of its recent magnitude"](
+/// Reference: ["RMSProp: Divide the gradient by a running average of its recent magnitude"](
```

Comment:
Since the paper calls it `rmsprop`, I think it's fine to leave it as is. Similarly, when we referred to the AdaDelta paper, we used `ADADELTA` instead of `AdaDelta` because the paper said so.